code/modules/clothing/under/costume.dm

```diff
@@ -85,10 +85,16 @@
 	female_sprite_flags = FEMALE_UNIFORM_TOP_ONLY
 	can_adjust = FALSE
 
-/obj/item/clothing/under/costume/kilt/highlander
+/obj/item/clothing/under/kilt_highlander
```

Comment:
remove it from the costume file if it's not a costume anymore

Reply:
though I do think it's silly to see you play whack-a-mole with costume subtypes, and think it would've been much smarter to use a white/black list